markdown/events/2023/bytesize_transcripts.md

```diff
@@ -23,3 +23,50 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[5:51](https://www.youtube.com/watch?v=amwwmFMwOYw&t=351)
+But even though Whisper is great, it is not perfect. I don't think any automated talk transcript ever will be perfect. The main things that we have to do is add timestamps so we have nice sections that belong together. But of course, also names, specifically names of people, but also of tools often get not identified correctly. You have to check and edit those. Specialized terminology is also not recognized because they are probably not in the library of Whisper. And also sometimes sentences are super long. It might be ellipsis or that someone had a thought, stopped in the thought and continued afterwards, which is totally fine if you're just listening to a person. But if you want to just read it, it's very difficult to understand. These kinds of things we have to manually change afterwards.
+
+[6:55](https://www.youtube.com/watch?v=amwwmFMwOYw&t=415)
+To give you a kind of an idea. Our most favorite words that are part of pretty much every bytesize talk, nf-core and Nextflow are very commonly misspelled. Nf-core very typically gets misspelled to NFL and NF4. I don't exactly know why, but in every third or so transcript I read those. And of course, you also have just some misspelling of nf-core itself. Sometimes it does pick it up and in very, very rare cases, it will also type it correctly. Nextflow, it also has diverse ways of how it can be written. In the latest one, I had it transcribed to "next floor". But then of course, there's just some random things that don't repeat. Like, "elusion" will be transcribed to "illusion", "iterations" to "situations". One of my favorites was "bioinformaticians" to "by partitions". Surprisingly, bioinformaticians, which is not that uncommon a word, I would say, gets transcribed a lot wrong. And you can imagine that if you have ribosomal RNA mistranscribed to rivals of RNA, the sentence will not make any sense. The handy overall summary can also become a handy oral summary, which would make sense, but which would change the meaning a bit. And just one other example, if you have a sentence like, "these processes take a sort of BAM from the samples", if you just read the sentence, I would have not guessed specifically what this would mean. Once I listened to the transcript, it turned out that it means "these processes take in a sorted BAM from SAMtools". This, I think, shows very clearly that manual work is necessary and that it's worth going through this and make these changes and not just rely on an automated transcript.
```

Comment:
How many trancriptions errors did you get from the usually badly transcripts phrases/words?

Reply:
elusion was wrong and "by partitions" was "bipartitions", while bioinformaticians was correct. of course nf-core and Nextflow where wrong, but it did get SAMtools! Also funny, "rivals of RNA" was transcribed to "ribosomal RNA". Admittedly, the recording was a bit shaky there.